.github/renovate.json5

```diff
@@ -0,0 +1,3 @@
+{
+  extends: ['github>seek-oss/rynovate'],
```

Comment:
Was this intentional? It's not mentioned in the PR Description and seems like a tack-on that could probably be a separate discussion.

Reply:
Ah yeah, I was gonna tack it on but I'll remove it if to unblock this